github/commands.yaml

```diff
@@ -7,6 +7,8 @@ commands:
     options:

```

Comment:
future idea: we should consider generating the commands configuration from a more concise grammar (like the syntax for the command) - this file is long and prone to error, and the semantic information can be encoded better (and more human friendly, eg: `command-name [-h host] [-r repo] ...`